### PR TITLE
[Feature] adding countries property in continent

### PR DESCRIPTION
**Pull request by buster95**
image

### CODE REVIEW

File: scrapers/getWorldometers.js [hunk 1]
```diff
@@ -34,13 +34,21 @@ const mapRows = (_, row) => {
...
+				delete countryInfo.country;
+				country.countryInfo = countryInfo;
+				break;
+			}
+			case 12:
+				country[columns[index - 1]] = cell.text();
+				break;
+
+			default:
+				country[columns[index - 1]] = parseInt(cell.text().replace(/(\n|,)/g, '')) || 0;
```
Comment (puf17640) [inline scrapers/getWorldometers.js:51]: - [ ] remove console.log

- [ ]  change switch to if
`if (index === 0) {
                                 const countryInfo = countryUtils.getCountryData(cell.text().replace(/(\n|,)/g, ''));
				country.country = countryInfo.country || cell.text().replace(/(\n|,)/g, '');
				delete countryInfo.country;
				country.countryInfo = countryInfo;
} else {
                                  const value = cell.text();
                                  country[columns[index - 1]] = isNan(value) ? value : (parseInt(value.replace(/(\n|,)/g, '')) || 0);
}`

File: routes/api_worldometers.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ const getAllData = async (key) => {
 
 const fixApostrophe = (country) => {
 	country.country = country.country.replace(/"/g, '\'');
+	delete country.continent;
```
Comment (puf17640) [inline routes/api_worldometers.js:24]: is this really needed? 
i think i would like to leave it in but leaving it in could lead to the problem of having "continent" as the equivalent to "country" but at the same time, a country object also has the property "continent"

ready for discussion

File: scrapers/getWorldometers.js (hunk 1 above)
Comment (buster95) [reply scrapers/getWorldometers.js:51]: Hey dude but switch is better than if, if setence evaluate one by one and switch evaluate once directly to the code block

File: routes/api_worldometers.js (hunk 2 above)
Comment (buster95) [reply routes/api_worldometers.js:24]: Idk, what do you think @ebwinters ?

File: scrapers/getWorldometers.js (hunk 1 above)
Comment (puf17640) [reply scrapers/getWorldometers.js:51]: kinda true yes, let's leave it

File: routes/api_worldometers.js (hunk 2 above)
Comment (ebwinters) [reply routes/api_worldometers.js:24]: We have the `country` field in `/countries` endpoint so I think we have to leave it for consistency.

**Comment by ebwinters** [comment]
This is all very good stuff, but I am planning to change a lot of things on master branch today so it is best left to be merged tomorrow or the next day. Sorry just a bunch of stuff happening today don't want to overwhelm and get this feature lost.

File: routes/api_worldometers.js (hunk 2 above)
Comment (buster95) [reply routes/api_worldometers.js:24]: I gonna leave that property

**Comment by buster95** [comment]
> This is all very good stuff, but I am planning to change a lot of things on master branch today so it is best left to be merged tomorrow or the next day. Sorry just a bunch of stuff happening today don't want to overwhelm and get this feature lost.

Hey Ethan do you leave this issues openened or will be closed?

**Comment by ebwinters** [comment]
> > This is all very good stuff, but I am planning to change a lot of things on master branch today so it is best left to be merged tomorrow or the next day. Sorry just a bunch of stuff happening today don't want to overwhelm and get this feature lost.
> 
> Hey Ethan do you leave this issues openened or will be closed?
There are no currently associated issues with this PR. As for the issues in @puf17640 comment, you should remove the commented `console.log` line and we agreed switch is better than if.

**Comment by buster95** [comment]
> > > This is all very good stuff, but I am planning to change a lot of things on master branch today so it is best left to be merged tomorrow or the next day. Sorry just a bunch of stuff happening today don't want to overwhelm and get this feature lost.
> > 
> > 
> > Hey Ethan do you leave this issues openened or will be closed?
> 
> There are no currently associated issues with this PR. As for the issues in @puf17640 comment, you should remove the commented `console.log` line and we agreed switch is better than if.

Ok I gonna add that changes, just lets me to finish lunch

**Comment by puf17640** [comment]
good work walter!!

**Pull request merged**